api/client.yaml

```diff
@@ -251,13 +251,53 @@ paths:
           description: Customer removed from customers
         '400':
           description: Failed to delete customer, see error(s)
+      summary: Delete Customer by ID
+      tags:
+        - Customers
```

Comment:
Is this valid? Can we put them under `delete:` like the other endpoints.

Reply:
It's valid but moved it up!